### ISSUE TITLE
Installer stop-all command no longer stops codewind containers (0.3.1 and master)

### ISSUE THREAD

**Issue by jopit** [eclipse/codewind#389]
It only stops the application containers, not the codewind ones. Likely cause is this PR: https://github.com/eclipse/codewind-installer/pull/103.  This is happening in both the 0.3.1 branch and in the master branch.

**Comment by johnmcollier** [comment]
Looking at this section: https://github.com/eclipse/codewind-installer/blob/845d7f544b60d73b002f1af2ae6d37a3583691bd/actions/stop-all.go#L23

```
        containerArr := [2]string{}
	containerArr[0] = "codewind-pfe"
	containerArr[1] = "codewind-performance"
	containerArr[0] = "cw-"
	containerArr[1] = "appsody"
```
The array entries for `codewind-pfe` and `codewind-performance` are being overwritten by `cw-` and `appsody`.

I think changing it to the following would fix things:
```
        containerArr := [4]string{}
	containerArr[0] = "codewind-pfe"
	containerArr[1] = "codewind-performance"
	containerArr[2] = "cw-"
	containerArr[3] = "appsody"
```

**Comment by makandre** [comment]
After above fix, there is still problem stopping appsody container.

It turns out that installer does `stop` then `remove` of a container.  But when you stop an appsody container, appsody itself will already remove the container.  So I believe installer's `remove` fails.

**Comment by jopit** [comment]
Right, the installer's stop of an appsody container fails like so:
```
$ ./codewind-installer-macos stop-all
Stopping Codewind and Project containers
{f7f67d88ff09c3c501d5ace5806902c811d707dd1c80579db304adcaf8aefd90 [/cw-appsodynode-a2b133c0-d01a-11e9-8da0-5bc35bec1f87] appsody/nodejs-express:0.2 sha256:6e0080881bae084863948cafd389aaca0075955d7acf7bd8c28798ee33ed32d0 /appsody/appsody-controller --mode=run 1567715594 [{0.0.0.0 3000 32783 tcp} {0.0.0.0 9229 32782 tcp}] 0 0 map[] running Up 46 seconds {codewind_network} 0xc0003aa008 [{bind  /Users/jsp/codewind-workspace/.extensions/codewind-appsody-extension/bin/appsody-controller /appsody/appsody-controller   true rprivate} {bind  /Users/jsp/codewind-workspace/appsodynode /project/user-app   true rprivate} {volume appsodynode-deps /var/lib/docker/volumes/appsodynode-deps/_data /project/user-app/node_modules local z true }]}
Stopping container  [/cw-appsodynode-a2b133c0-d01a-11e9-8da0-5bc35bec1f87] ... 
2019/09/05 16:34:01 CONTAINER_ERROR[108]: Error response from daemon: removal of container f7f67d88ff09c3c501d5ace5806902c811d707dd1c80579db304adcaf8aefd90 is already in progress. 
```

**Comment by johnmcollier** [comment]
@makandre @jopit Would merging https://github.com/eclipse/codewind-installer/pull/106 address the issue with appsody?

**Comment by makandre** [comment]
Don't believe so as that is for removing images

**Comment by tobespc** [comment]
Im looking at this asap

**Comment by jagraj** [comment]
I tried to stop codewind and got into this failure...

```
There was a problem trying to stop Codewind: 2019/09/06 09:17:07 CONTAINER_ERROR[108]: Error response from daemon: No such container: eedd171d948ed7f8d0f58289c13739ffd3423477d2dcbcb15a65f64ecd2c3f65. 
```

**Comment by tobespc** [comment]
changes are all in and merged to master and 0.3.1 of the installer
